config/locales/en.yml

```diff
@@ -538,6 +538,7 @@ en:
     timeline: Assignment Details
     timeline_enabled: Timeline enabled
     timeline_link: Timeline
+    timeline_tooltip: Timeline help you to keep track of your course by breaking the course duration in weeks which have proper agendas and tasks associated with them.
```

Comment:
The only change I would make is a grammatical one: I would say "...into weeks with agendas and tasks associated with them." It's tighter as a whole and "into" is more clear because, for example, you can break something into pieces but don't break something in pieces. Just a suggestion! @psinghal20